Geometry/TrackerCommonData/plugins/DDTrackerIrregularRingAlgo.cc

```diff
@@ -0,0 +1,280 @@
+///////////////////////////////////////////////////////////////////////////////
+// File: DDTrackerIrregularRingAlgo.cc
+// Description:  Tilts and positions n copies of a module at prescribed phi
+// values within a ring. The module can also be flipped if requested.
+///////////////////////////////////////////////////////////////////////////////
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DetectorDescription/Core/interface/DDCurrentNamespace.h"
+#include "DetectorDescription/Core/interface/DDSplit.h"
+#include "DetectorDescription/Core/interface/DDRotationMatrix.h"
+#include "DetectorDescription/Core/interface/DDTransform.h"
+#include "DetectorDescription/Core/interface/DDTypes.h"
+#include "DetectorDescription/Core/interface/DDAlgorithm.h"
+#include "DetectorDescription/Core/interface/DDAlgorithmFactory.h"
+#include "CLHEP/Units/GlobalPhysicalConstants.h"
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
Please delete these two CLHEP includes. Switch to:
```
#include "DataFormats/Math/interface/angle_units.h"
using angle_units::operators;
```
Make changes like:
`90. * CLHEP::deg` to `90._deg`
`value / CLHEP::deg` to `convertRadToDeg(value)`